src/main/java/com/fasterxml/jackson/databind/ser/PropertyBuilder.java

```diff
@@ -56,8 +56,9 @@ public PropertyBuilder(SerializationConfig config, BeanDescription beanDesc)
         // 08-Sep-2016, tatu: This gets tricky, with 3 levels of definitions:
         //  (a) global default inclusion
         //  (b) per-type default inclusion (from annotation or config overrides;
-        //     latter having precedence)
-        //  Cc) per-property override
+        //     config override having precedence)
+        //  (c) per-property override (from annotation or config overrides;
```

Comment:
This is sort of correct but maybe misleading, as config override is for type-of-property, but not for specific property. Not sure how to properly explain the distinction.